portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,18 @@
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login-url")
+public class LoginServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html");
+    String loginUrl =
+        UserServiceFactory.getUserService().createLoginURL(/* urlToRedirectToAfterLogin= */ "/");
+    response.getWriter().println(loginUrl);
```

Comment:
Small thing: I think it would make sense to group `response.setContentType` and `response.getWriter()` in a consistent way between this Servlet, LoginStatusServlet, and future servlets - either in this way, or right next to each other (@LoginStatusServlet:37)

Reply:
Done - noted will keep it consistent in the future.